engine/dirhandler.go

```diff
@@ -137,6 +137,20 @@ func (ac *Config) DirPage(w http.ResponseWriter, req *http.Request, rootdir, dir
 		}
 	}
 
+	// Serve handler.lua, if found in ancestors
+	var ancestor string
+	ancestor = filepath.Dir(dirname)
+	for true {
```

Comment:
As a habit, I always use a for loop with a large number instead of `for true` whenever possible.

What do you think about `for i := 0; i < 1000; i++` instead, just to have more defensive coding?